requirements.txt

```diff
@@ -46,3 +46,6 @@ bip-utils==2.9.3
 
 #constraints
 pycparser<=2.17 # ugly -- to avoid https://github.com/eliben/pycparser/pull/198
+
+# used for the packaging script that requires to have a version pinned for it
+ckzg==1.0.1
```

Comment:
Is this used in production or packaging? If only packaging then this should go to requirements_packaging.txt